doc/userguide/rules/rfb-keywords.rst

```diff
@@ -25,9 +25,12 @@ rfb.secresult
 
 Match on the value of the RFB security result, e.g. ``ok``, ``fail``, ``toomany`` or ``unknown``.
 
+rfb.secresult uses an :ref:`unsigned 32-bit integer <rules-integer-keywords>`.
+
 Examples::
 
   rfb.secresult: ok;
+  rfb.secresult: !0;
```

Comment:
Considering this is new, I think it'd be worth adding a new rule to https://github.com/OISF/suricata-verify/blob/master/tests/rfb-rules/test.rules to include this format. (I can do that, if the idea is agreed upon :) )

Reply:
added in https://github.com/OISF/suricata-verify/pull/1985